blocks/cards/cards.css

```diff
@@ -176,7 +175,7 @@ main .cards.machine-parts .cards-card-body a > strong {
 main .cards.machine-parts ul > li {
   padding: 0;
   box-shadow: none;
-  background: var(--transparent-blue-light-color);
+  /*background: var(--transparent-blue-light-color);*/
```

Comment:
You can remove instead of commenting as git history would alway have it for reference.